### ISSUE TITLE
Disk Formatting doesn't happen after adding the new drives, and also no option for C drive vm customization with the Kitchen Vcenter

### ISSUE THREAD

**Issue by prakash1243** [chef/kitchen-vcenter#168]
### Versions:

* Version of kitchen-vcenter:  2.11.9
* Version of test-kitchen: 
* Version of chef:

### Platform Details

* Version of vCenter: 6.7
* Version of ESXi:

### Expected Result:
 We should have the option to customize the disk size on C drive and also adding new drives on vm_customization should create drives on the virtual machine

### Actual Result:
 I actually see the drives being added on the VM on vcenter, but it doesn't appear to be on the virtual machine

**Comment by tecracer-theinen** [comment]
A note on the formatting part: The feature was explicitly added for a customer who simulates physical machines and formatting the new drives was/is a part of that (e.g. for SQL servers with different block sizes). I would keep that as a default.

In addition, OS-level interaction like formatting will need supplying OS credentials and use of the GOM tools (like for active discovery, so they're already in there). I see the use case but am skeptical if this level of complexity is needed. Customization makes the process slower and if it's a regular thing, you likely want an adjusted template with multiple, pre-formatted disks anyway in my experience.

Totally agree on the C-drive customization though ;-)

**Comment by prakash1243** [comment]
I agree with the pre-defined template with adjusted formatted disks :-)  looking forward for the customization on the C drive feature. Thanks for your response !
